### ISSUE TITLE
Error with local storage on second launch since v1.5.7

### ISSUE THREAD

**Issue by wedgybo** [BranchMetrics/Web-SDK#75]
I've raised an issue through the support form, but this should help out a little more.

I've seeing issues when v1.5.7 is attempting to load data from local storage. The key causing the issue is this.

`BRANCH_WEBSDK_KEYdata : {"+is_first_session":false,"+clicked_branch_link":false}`

It's attempting to parse this key twice for some reason and then validate the string version of the object which is '[Object object]' and fails. It's difficult to understand why on the minified code however.

```
Uncaught Error: Invalid JSON string: [object Object]
  m @ branch-v1.5.7.min.js:2
  d @ branch-v1.5.7.min.js:51
  (anonymous function) @ branch-v1.5.7.min.js:52
  (anonymous function) @ branch-v1.5.7.min.js:48
  a @ branch-v1.5.7.min.js:2
  (anonymous function) @ branch-v1.5.7.min.js:2
  (anonymous function) @ branch-v1.5.7.min.js:47
  (anonymous function) @ branch-v1.5.7.min.js:60
  (anonymous function) @ branch-v1.5.7.min.js:60
```

I've tried the version from bower as well which gives us a better stack trace, but still the functions are to confusing to offer any real help in how to fix it.

```
Error: Invalid JSON string: [object Object]
    at Error (native)
    at Object.goog.json.parse (http://localhost:9000/components/branch-sdk/dist/web/build.js:638:9)
    at Branch.init.f (http://localhost:9000/components/branch-sdk/dist/web/build.js:1440:49)
    at null.<anonymous> (http://localhost:9000/components/branch-sdk/dist/web/build.js:1451:5)
    at Array.<anonymous> (http://localhost:9000/components/branch-sdk/dist/web/build.js:1366:9)
    at b (http://localhost:9000/components/branch-sdk/dist/web/build.js:613:11)
    at Queue [as _queue] (http://localhost:9000/components/branch-sdk/dist/web/build.js:621:22)
    at wrap (http://localhost:9000/components/branch-sdk/dist/web/build.js:1346:7)
    at BranchProvider.self.setAppKey (http://localhost:9000/services/branch/branch.js:21:16)
    at http://localhost:9000/app.js:116:21
```

**Comment by playtogga** [comment]
Having the same issue! Hoping for a quick solution.

**Comment by dmitrig01** [comment]
ack! thanks for catching this guys. a fix is on the way.

**Comment by scotthasbrouck** [comment]
Hey @playtogga and @wedgybo Just pushed a fix for this as version 1.5.8

Thanks for catching this!

**Comment by wedgybo** [comment]
:+1: Will check it out tomorrow
